gsa/src/web/components/menu/menu.js

```diff
@@ -114,13 +116,24 @@ const MenuList = styled.ul`
   }
 `;
 
+const checkChild = child => {
+  if (child.type === MenuSection && isArray(child.props.children)) {
+    return child.props.children.find(chil => chil !== false);
+  } else if (child.type === MenuSection && isDefined(child.props.children)) {
+    return child.props.children;
+  }
+  return child;
+};
+
 const Menu = ({children, title, to, ...props}) => {
   let link;
   children = React.Children.toArray(children).filter(hasValue);
+
   if (isDefined(to)) {
     link = <Link to={to}>{title}</Link>;
   } else if (isDefined(children) && children.length > 0) {
-    const [child] = children;
+    let [child] = children;
+    child = checkChild(child);
```

Comment:
IMHO the function name checkChild is a bit misplaced. I do expect either to return a boolean or to raise an exception for a check named function. I guess
```javascript
getFirstMenuEntry(children)
```
would fit?